docs/docusaurus/versioned_sidebars/version-0.17-sidebars.json

```diff
@@ -1,227 +1,6 @@
 {
   "docs": [
     "intro",
-    {
```

Comment:
No, this in particular is because we don't want to version GX cloud documentation, so the idea is to always remove it from all the versions that are not the current one (in this case i'm deleting the one in 0.17 so when you click on GX Cloud tab it always show the one from version 0.18 but we should keep removing them when releasing newer versions, so defintely is not a very maintainable solution in the long term